tests/test_iterable_validator.py

```diff
@@ -88,6 +98,7 @@ def test_mixed(self):
         self.assertTrue(validator({1: 'pyvalid', 2: 'cython'}))  # Dictionary
         self.assertTrue(validator((1, 3, 25, 120)))  # Set
         self.assertFalse(validator(8))  # Integer
+        self.assertFalse(None)
```

Comment:
@uzumaxy 
Why is this assert for None required in all the test cases? As we are checking for the type of val (here iterable) and not its elements, isn't is obvious that `self.assertFalse(validator(None))` will pass in all other test cases if it passes in just `test_mixed` test case?